reclama/templates/index.html

```diff
@@ -14,4 +14,16 @@
       </div>
     </a>
   {% endfor %}
+
+  <div class="archived">Archived events</div>
```

Comment:
Please check if archived events exist. At the moment if no archived events exist it just shows the title.